memex/src/test/java/com/johnlpage/memex/cucumber/steps/InspectionSteps.java

```
@@ -47,18 +62,70 @@ public Boolean bool(String bool){
+    public void theFollowingVehicleInspectionsInRangeDoesNotExist(long startId, long endId) {
+        Query query = new Query();
+        query.addCriteria(Criteria.where("_id").gte(startId).lte(endId));
+        mongoTemplate.remove(query, VehicleInspection.class);
     }
 
-    @Given("the following vehicle inspections exist and have historical data as of {string}:")
-    public void givenVehicleInspectionsExist(String date, DataTable dataTable) {
-        // test assumes it exists however it would be better to create it here
+    @Given("the vehicle inspection with id: {long} do not exist:")
+    public void theFollowingVehicleInspectionDoesNotExist(long startId) {
+        Query query = new Query();
+        query.addCriteria(Criteria.where("_id").is(startId));
+        mongoTemplate.remove(query, VehicleInspection.class);
     }
 
     @Given("the following vehicle inspections do not exist:")
-    public void givenVehicleInspectionsDoNotExist(DataTable dataTable) {
-        // test assumes it does not exist however it would be better to delete it here
+    public void givenTheFollowingVehicleInspectionsDoNotExist(DataTable dataTable) {
+        for (Map<String, String> row : dataTable.asMaps()) {
+
+            if (row.size() != 1) {
+                throw new IllegalArgumentException("Only one column per row is supported in this step.");
+            }
+
+            String key = row.keySet().iterator().next();
+            String value = row.get(key);
+
+            boolean idInQueryExists = false;
+            Query query = switch (key) {
+                case "testid" -> {
+                    long testid = Long.parseLong(value);
+                    if (testid >= idStart && testid <= idEnd) {
+                        idInQueryExists = true;
+                        yield Query.query(Criteria.where("_id").is(testid));
+                    } else {
+                        yield null;
```

test data range violation should not be quiet, it will be a nightmare to debug. Please throw an Exception